src/main/webapp/app/therapeuticareas/views/therapeutic.area.questionnaire.view.htm

```diff
@@ -103,119 +37,57 @@
-                        </button>
-                    </div>
-                </div>
-                <div class="clear"></div>
-
-            </div>
-
-        </div>
-        </div>
-        -->
-    </form>
-</div>
 
 
-
-<div class="clear"></div>
-
-    <!--
-    <div as-sortable="sortableOptions" ng-model="questionnaireEntries">
-            <div ng-repeat="item in questionnaireEntries" as-sortable-item class="as-sortable-item">
-    -->            
-    <!--
-                <div as-sortable-item-handle class="as-sortable-item-handle" style="height:50px">
-                    <span data-ng-bind="item.question.questionText"></span>
-                </div>
-
-                
-                <div as-sortable="sortableOptions" ng-model="item.answers._embeddedItems">
-                    <div ng-repeat="answer in item.answers._embeddedItems" as-sortable-item class="as-sortable-item">
-                        <div as-sortable-item-handle>
-                            <span data-ng-bind="answer.answerText"></span>
+                    <!--
+                    <div data-as-sortable="sortableOptions" data-ng-model="questionnaireEntry.answers._embeddedItems">
+                        <div data-ng-repeat="answer in questionnaireEntry.answers._embeddedItems" data-as-sortable-item class="as-sortable-item">
+                            <div data-as-sortable-item-handle>
+                                <span data-ng-bind="answer.answerText"></span>
+                            </div>
                         </div>
                     </div>
-                </div>
-                -->
+                    -->
+
 
-                <!--
-                <div as-sortable="sortableOptions" ng-model="questionnaireEntries">
-                    <div ng-repeat="item in questionnaireEntries" as-sortable-item class="as-sortable-item">
-                        <div as-sortable-item-handle>
-                            <span data-ng-bind="item.question.questionText"></span>
-                        </div>
-                    </div>
                 </div>
-                -->
-                
-                <!--
-                <div as-sortable="sortableOptions" ng-model="questionnaireEntry.answers._embeddedItems">
-                    <div ng-repeat="answer in questionnaireEntry.answers._embeddedItems | orderBy:'id'" as-sortable-item class="as-sortable-item">
-                       <div as-sortable-item-handle class="as-sortable-item-handle" style="height:50px">
-                            <span data-ng-bind="answer.answerText"></span>
-                        </div>
+
+                <div class="answers" data-ng-if="questionnaireEntry.answers._embeddedItems.length>=5">
+                    <div class="option-list">
+                        <button id="btn_deleteAnswer{{questionnaireEntry.answers._embeddedItems.id}}" class="left btn btn-remove" type="button" title="Delete"
```

Comment:
same as above here, answer is available to the ng-click so should be available for the ng-attr-id